eq-author/src/App/history/HistoryItem.js

```diff
@@ -50,7 +59,7 @@ const HistoryItem = ({
   return (
     <StyledItem>
       <QuestionnaireTitle>
-        {questionnaireTitle} - <strong>{publishStatus}</strong>
+        {questionnaireTitle} - <strong>{formatText(publishStatus)}</strong>
```

Comment:
```suggestion
        {questionnaireTitle} - <strong>{translations[publishStatus]}</strong>
```

Or is there any cases where the publishStatus wont match any of the keys in the `translations`?